File: courses/POO/Curso Mello (2009)/Archives/COMP07 Files/aulas/ex36/c++/Teste.cc
#include "Teste.h"

int main(int argc, char *argv[]) {
	Teste *t = new Teste();
	cout << t->getA() << endl;

	int a = 25;
	*t = a;
	cout << t->getA() << endl;

	Teste *t2 = new Teste(15);
	cout << t2->getA() << endl;

	cout << t->soma(1,1) << endl;
	cout << t->soma(1,1.5) << endl;

	return 0;
}

File: courses/POO/Curso Mello (2009)/Archives/COMP07 Files/exemplos/ex31/Teste.java
public class Teste {
	int x, y;

	public Teste(int x, int y) {
		this.x = x;
		this.y = y;
	}

	public void imprimir() {
		System.out.println(this.x+" "+this.y);
		//System.out.println(x+" "+y);
	}

	public static void main(String args[]) {
		Teste t = new Teste(5,4);
		t.imprimir();
	}
}
